packages/.snyk

```diff
@@ -0,0 +1,16 @@
+ # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities
+version: v1.25.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+    SNYK-JS-IP-7148531:
```

Comment:
Do you think we will end up with dozens of these entries for the `0.0.0-use.local` issue?

Reply:
For all the closed PRs only these two snyk codes were reported that is why I added them to be ignored:
![Screenshot 2024-07-31 at 14 29 11](https://github.com/user-attachments/assets/b4caef36-50ff-4555-95c4-a951298e351a)